src/test/kotlin/com/anaplan/engineering/vdmgradleplugin/PomRewriterTest.kt

```diff
@@ -49,7 +49,10 @@ class PomRewriterTest {
         Files.copy(sourceFileOrig, sourceFileCopy)
         makeChanges(PomRewriter(sourceFileCopy.toFile()))
         val expectedFile = Paths.get(javaClass.getResource("$resourceSuffix-expected.xml").toURI())
-        Assert.assertEquals(expectedFile.toFile().readText(), sourceFileCopy.toFile().readText())
+        Assert.assertEquals(
+            PomRewriter(expectedFile.toFile()).readDocument().toString(),
```

Comment:
This feels like it should be extracted to an XML utility; feels odd to be using `PomRewriter` to create the `Document` like this